src/components/Main.js

```diff
@@ -71,7 +86,8 @@ var Main = React.createClass({
             <div className="tab">SCENEGRAPH</div>
             <Scenegraph scene={scene}/>
             <div className="scenegraph-bottom">
-              <a href="#" onClick={this.deleteEntity} className="button fa fa-trash-o"></a>
+              <a href="#" title="Delete selected entity" onClick={this.deleteEntity} className="button fa fa-trash-o"></a>
+              <a href="#" title="Add new entity" onClick={this.addEntity} className="button fa fa-plus"></a>
```

Comment:
I added it to the top of the scenegraph header instead in my PR.  I was thinking that we could add the delete icon inline with the entity name:

[entity name][type]    [delete][visibility toggle]